fdbserver/DataDistribution.actor.cpp

```diff
@@ -536,6 +566,167 @@ struct DataDistributor : NonCopyable, ReferenceCounted<DataDistributor> {
 	}
 };
 
+ACTOR Future<Void> monitorPhysicalShardStatus(Database cx, Reference<PhysicalShardCollection> self) {
+	ASSERT(CLIENT_KNOBS->DD_PHYSICAL_SHARD_CORE);
+	loop {
+		wait(delay(SERVER_KNOBS->ROCKSDB_METRICS_DELAY * 2));
```

Comment:
why `*2`?

Reply:
It is a casual number. I create a knob for this.